<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="Preetams-MacBook-Air.local" failures="0" tests="9" name="com.tests.ValidationEpic1" time="106.675" errors="1" timestamp="2023-09-07T21:00:35 IST" skipped="8">
  <testcase classname="com.tests.ValidationEpic1" name="searchProduct" time="106.675">
    <error message="For input string: &quot;&quot;" type="java.lang.NumberFormatException">
      <![CDATA[java.lang.NumberFormatException: For input string: ""
at java.base/java.lang.NumberFormatException.forInputString(NumberFormatException.java:68)
at java.base/java.lang.Integer.parseInt(Integer.java:662)
at java.base/java.lang.Integer.parseInt(Integer.java:770)
at com.pages.ProductSearchResult.lambda$0(ProductSearchResult.java:54)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at com.pages.ProductSearchResult.validateProductResultSorting(ProductSearchResult.java:54)
at com.tests.ValidationEpic1.searchProduct(ValidationEpic1.java:138)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:64)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:564)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130)
at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630)
at java.base/java.lang.Thread.run(Thread.java:832)
]]>
    </error>
  </testcase> <!-- searchProduct -->
  <system-out/>
  <testcase classname="com.tests.ValidationEpic1" name="searchFlights4">
    <skipped/>
  </testcase> <!-- searchFlights4 -->
  <system-out/>
  <testcase classname="com.tests.ValidationEpic1" name="searchFlights2">
    <skipped/>
  </testcase> <!-- searchFlights2 -->
  <system-out/>
  <testcase classname="com.tests.ValidationEpic1" name="searchFlights">
    <skipped/>
  </testcase> <!-- searchFlights -->
  <system-out/>
  <testcase classname="com.tests.ValidationEpic1" name="validateOfferesIsSelectedNegative">
    <skipped/>
  </testcase> <!-- validateOfferesIsSelectedNegative -->
  <system-out/>
  <testcase classname="com.tests.ValidationEpic1" name="validateBrokenIsSelected">
    <skipped/>
  </testcase> <!-- validateBrokenIsSelected -->
  <system-out/>
  <testcase classname="com.tests.ValidationEpic1" name="validateHotelIsSelected2">
    <skipped/>
  </testcase> <!-- validateHotelIsSelected2 -->
  <system-out/>
  <testcase classname="com.tests.ValidationEpic1" name="validateOfferesIsSelected">
    <skipped/>
  </testcase> <!-- validateOfferesIsSelected -->
  <system-out/>
  <testcase classname="com.tests.ValidationEpic1" name="searchFlights3">
    <skipped/>
  </testcase> <!-- searchFlights3 -->
  <system-out/>
</testsuite> <!-- com.tests.ValidationEpic1 -->
